<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="drillAV">
    <link name="link">
      <inertial>
        <pose>-0.00637 -0.008 0.13254 0 0 0</pose>
        <inertia>
          <ixx>0.01331127</ixx>
          <ixy>-0.00030365</ixy>
          <ixz>-0.00034148</ixz>
          <iyy>0.01157659</iyy>
          <iyz>0.00088073</iyz>
          <izz>0.00378028</izz>
        </inertia>
        <mass>1.50251902</mass>
      </inertial>
      <collision name="collision">
        <pose>0 0 -0.09 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://drillAV/meshes/cordless_drill.stl</uri>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
              <soft_erp>0.5</soft_erp>
              <soft_cfm>0.001</soft_cfm>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>10000.0</mu>
              <mu2>10000.0</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
        <pose>0 0 -0.09 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://drillAV/meshes/cordless_drill.dae</uri>
          </mesh>
        </geometry>
    	<material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>
